src/history/index.js

```diff
@@ -6,25 +6,46 @@ import * as git from './git.js';
 
 export const ROOT_DIRECTORY = path.resolve() + '/data';
 export const RAW_DIRECTORY = `${ROOT_DIRECTORY}/raw`;
+export const SANITIZED_DIRECTORY = `${ROOT_DIRECTORY}/sanitized`;
 
 export async function persistRaw(serviceProviderId, policyType, fileContent) {
-  await storeRaw(serviceProviderId, policyType, fileContent);
-  return await commitRaw(serviceProviderId, policyType);
+  await persist({
+    serviceProviderId, 
+    policyType, 
+    fileContent,
+    isSanitized: false
+  });
 }
 
-export async function storeRaw(serviceProviderId, policyType, fileContent) {
-  const dir = `${RAW_DIRECTORY}/${serviceProviderId}`;
+export async function persistSanitized(serviceProviderId, policyType, fileContent) {
+  await persist({
+    serviceProviderId, 
+    policyType, 
+    fileContent,
+    isSanitized: true
+  });
+}
+
+export async function persist({ serviceProviderId, policyType, fileContent, isSanitized }) {
+  await store({ serviceProviderId, policyType, fileContent, isSanitized });
+  return await commit({ serviceProviderId, policyType, isSanitized });
+}
+
+export async function store({ serviceProviderId, policyType, fileContent, isSanitized }) {
+const directory = `${isSanitized ? SANITIZED_DIRECTORY : RAW_DIRECTORY}/${serviceProviderId}`;
 
-  if (!fsApi.existsSync(dir)){
-    fsApi.mkdirSync(dir);
+if (!fsApi.existsSync(directory)){
+    fsApi.mkdirSync(directory);
   }
 
-  return fs.writeFile(`${dir}/${policyType}.html`, fileContent);
+  return fs.writeFile(`${directory}/${policyType}.${isSanitized ? 'md' : 'html'}`, fileContent);
 }
 
-export async function commitRaw(serviceProviderId, policyType) {
+export async function commit({ serviceProviderId, policyType, isSanitized }) {
```

Comment:
`isSanitized` has a default value of `undefined` which is falsy. I would have preferred to make that distinction super-clear through the function name (`storeRaw` / `storeSanitized`). If we want to go the way of passing a single options object, then I believe we should validate in some way that option.

Unless of course this is exported only for testing purposes…? Let's have a chat around this 🙂